install_config/aggregate_logging.adoc

```diff
@@ -2054,6 +2054,244 @@ $ oc exec <$espod> -- es_load_kibana_ui_objects <user-name>
 
 Where `$espod` is the name of any one of your Elasticsearch pods.
 
+[[kibana-visualizations-custom]]
+*Adding Custom Fields to Kibana Visualize*
+
+If your {product-title} cluster is configured with custom logging fields
+that are not are not defined in the `.operations.\*` or the `project.*` 
+index, you cannot create visualizations with these fields. 
+
+However, if you if you use the `MERGE_JSON_LOG` feature (`MERGE_JSON_LOG=true`) 
+to generate JSON format logs, you can use these custom fields in 
+Kibana Visualize by adding the fields to the Elasticsearch index patterns and to Kibana. 
+
+To add custom fields to Kibana Visualize:
+
+. Add custom fields to an Elasticsearch index template:
```

Comment:
We may need to add a note here: these fields will only be applied to the indices created after the template is updated.